emukit/model_wrappers/gpy_model_wrappers.py

```diff
@@ -48,7 +53,7 @@ def optimize(self):
         """
         Optimizes model hyper-parameters
         """
-        self.model.optimize()
+        self.model.optimize_restarts(self.n_restarts, robust=True)
```

Comment:
Is optimize() and optimize_restarts(1) the same? It might differ in case one is deterministic and the other one random.

Reply:
Yep, it's the same. 

[Here](https://github.com/sods/paramz/blob/master/paramz/model.py#L170) is the code that does the randomization, it only randomizes before the second and subsequent restarts.

Reply:
👍